index.html

```diff
@@ -59,138 +62,173 @@
+        <span class="es2" ><P style="color:aqua">  Elasticsearch 2.X</P></span>
+        <span class="es5 " ><P style="color:aqua"> Elasticsearch 5.X</P></span>
+        <span class="es6" ><P style="color:aqua">Elasticsearch 6.X</P> </span>
+        <i class="dropdown icon"></i>
+        <div class="menu">
+          <div class="item es2 es5 es6"><a href="#es1">Elasticsearch 1.X</a></div>
+          <div class="item es1 es5 es6"><a href="#es2">Elasticsearch 2.X</a></div>
+          <div class="item es1 es2 es6"><a href="#es5">Elasticsearch 5.X</a></div>
+          <div class="item es1 es2 es5"><a href="#es6">Elasticsearch 6.X</a></div>
         </div>
       </div>
     </div>
+  </div>
 
-    <div class="ui text container language-json">
+  <div class="ui text container language-json">
 
-      <div class="es1 ui secondary orange segment">
-        <p>Elasticsearch 1.7 End Of Life. Consider upgrading.
-        <a href="https://www.elastic.co/support/eol">More information about supported versions</a>.</p>
-      </div>
+    <div class="es1 ui secondary orange segment">
+      <p>Elasticsearch 1.7 End Of Life. Consider upgrading.
+        <a href="https://www.elastic.co/support/eol"  style="color:limegreen">More information about supported versions</a>.</p>
+    </div>
 
-      <div class="es2 ui secondary orange segment">
-        <p>Elasticsearch 2.4 will soon be End Of Life. Consider upgrading.
-        <a href="https://www.elastic.co/support/eol">More information about supported versions</a>.</p>
-      </div>
+    <div class="es2 ui secondary orange segment">
+      <p>Elasticsearch 2.4 will soon be End Of Life. Consider upgrading.
+        <a href="https://www.elastic.co/support/eol" style="color:limegreen">More information about supported versions</a>.</p>
+    </div>
 
-      <a name="links"></a>
-      <h2 class="ui dividing header">Links</h2>
+    <a name="links"></a>
+    <h2 class="ui dividing header" style="color:orangered" > Links</h2>
 
 
-        <p class="es1">
-          First thing, forget about your <code>curl</code> calls and <strong>install <a
-            href="https://chrome.google.com/webstore/detail/sense-beta/lhjgkmllcaadmopgmanpapmpjgmfcfig?hl=en">the Sense extension</a> please !</strong>
-        </p>
-        <p class="es2">
-          First thing, forget about your <code>curl</code> calls and <strong>install <a href="https://www.elastic.co/guide/en/sense/current/installing.html">Sense</a> please !</strong>
-        </p>
-        <p class="es5 es6">
-          First thing, forget about your <code>curl</code> calls and <strong>install <a href="https://www.elastic.co/products/kibana">Kibana</a> please !</strong>
-        </p>
+    <p class="es1">
+      First thing, forget about your <code>curl</code> calls and <strong>install <a href="https://chrome.google.com/webstore/detail/sense-beta/lhjgkmllcaadmopgmanpapmpjgmfcfig?hl=en" style="color:limegreen">the
+          Sense extension</a> please !</strong>
```

Comment:
The new line here and in other places maybe make it easier to read on your editor / system, but now it cut the sentence in the middle with a strange indentation. Every line in the Cheatsheet change because of this ; which is really only a matter of personal taste. Could you please get back to the original?